lib/v2/datasource/remote/model/token_model.dart

```diff
@@ -1,30 +1,64 @@
+
 class TokenModel {
   final String chainName;
   final String contract;
   final String symbol;
   final String name;
+  final String backgroundImage;
+  final String logo;
+  final String balanceSubTitle;
+
   const TokenModel({
     required this.chainName,
     required this.contract,
     required this.symbol,
     required this.name,
+    required this.backgroundImage,
+    required this.logo,
+    required this.balanceSubTitle,
     });
+
+  @override
+  bool operator ==(Object other) =>
```

Comment:
i think you can extend equatable class to avoid doing this. 
@RaulUrtecho  ?

Reply:
Equatable can be used as usual. 

But if I'm honest, if it's a class that I know I don't use anywhere in my app to compare its instances by value, 
I don't see the need to use Equatbale or this.

Reply:
Cool. Done!